### PR TITLE
Add Glitch

### PR DESCRIPTION
**Pull request by matthewdias**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

**Review by fauxpark — CHANGES_REQUESTED** [review]
Duplicate board again.

**Review by fauxpark — CHANGES_REQUESTED** [review]
Missing an info.json.

**Review by drashna — APPROVED** [review]

File: keyboards/p3d/glitch/info.json [hunk 1]
```diff
@@ -0,0 +1,300 @@
+{
+    "keyboard_name": "Glitch",
+    "url": "https://p3dstore.com/products/p3d-glitch-group-buy",
+    "maintainer": "matthewdias",
+    "layouts": {
+        "LAYOUT": {
```
Comment (fauxpark) [inline keyboards/p3d/glitch/info.json:6]: Layout name does not match glitch.h.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/p3d/glitch/readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Glitch
+
+Hotswap RGB Alice-style with OLED and Rotary Knob
+
+-   Keyboard Maintainer: matthewdias
+-   Hardware Supported: Glitch PCB
+-   Hardware Availability: https://p3dstore.com
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/p3d/glitch/readme.md:9]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — APPROVED** [review]

File: keyboards/p3d/glitch/readme.md (hunk 2 above)
Comment (matthewdias) [reply keyboards/p3d/glitch/readme.md:9]: Done!

**Review by drashna — APPROVED** [review]

**Pull request merged**